internal/platform/cpuid_amd64.go

```diff
@@ -0,0 +1,73 @@
+package platform
+
+const (
+	CpuFeatureSSE3   = uint64(1)
```

Comment:
Let’s put the comment as much as possible on exported comets even its under internal. That helps us in the future